_posts/2022-04-20-WildFly-s2i-v2-Released.adoc

```diff
@@ -0,0 +1,232 @@
+
+=== Release of the new images
+
+Images release cadence is now un-related to the WildFly server release cadence. Images have their own lifecycle. New releases will be done to address fixes and new features.
+Released images are deployed in the link:http://quay.io/wildfly[quay.io/wildfly organization]
+For the one that love to be on the bleeding edge we are constantly deploying images under development to the link:http://quay.io/wildfly-snapshots[quay.io/wildfly-snapshots organization].
+
+=== Using the JDK 17 S2I builder image
+
+Applications are built and run similarly on JDK11 and JDK17. For JDK17 just make sure to have your maven 
+projects to use `3.3.2` version for the link:https://maven.apache.org/plugins/maven-war-plugin/[Maven war plugin]. 
+This link:https://github.com/wildfly/wildfly-s2i/tree/main/examples/postgresql-jakarta/README.md[example] shows how to configure the Maven war plugin version. 
+
+[NOTE]
+====
+When using the JDK17 image, you need to set the following env variable to workaround link:https://issues.redhat.com/browse/WFCORE-5835[WFCORE-5835]:
+```
+JAVA_OPTS_APPEND=--add-exports=jdk.naming.dns/com.sun.jndi.dns=ALL-UNNAMED
+```
+====
+
+=== Multi arch images preview
+
+We have released preview of multi arch images (linux/amd64 and linux/arm64). Longer term we plan 
+to make our official images multi arch and stop releasing the preview images, but we are not yet there.
+
+Multi arch preview images:
+
+* link:https://quay.io/repository/wildfly-snapshots/wildfly-s2i-jdk11-multi-arch[https://quay.io/repository/wildfly-snapshots/wildfly-s2i-jdk11-multi-arch] preview multi arch JDK11 S2I builder image.
+
+* link:https://quay.io/repository/wildfly-snapshots/wildfly-s2i-jdk17-multi-arch[https://quay.io/repository/wildfly-snapshots/wildfly-s2i-jdk17-multi-arch] preview multi arch JDK17 S2I builder image.
+
+* link:https://quay.io/repository/wildfly-snapshots/wildfly-runtime-jdk11-multi-arch[https://quay.io/repository/wildfly-snapshots/wildfly-runtime-jdk11-multi-arch] preview JDK11 multi arch runtime image.
+
+* link:https://quay.io/repository/wildfly-snapshots/wildfly-runtime-jdk17-multi-arch[https://quay.io/repository/wildfly-snapshots/wildfly-runtime-jdk17-multi-arch] preview JDK17 multi arch runtime image.
+
+== New s2i build workflow
+
+In order to create a server to be installed into the WildFly image we are relaying on the 
+link:https://github.com/wildfly/wildfly-maven-plugin/[WildFly Maven plugin 3.0.0.Final] that can now provision a fully configured server containing your deployment. 
+The WildFly Maven plugin `3.0.0.Final` has been evolved with some new goals to provision, configure, and package the server and the deployment in one step. 
+When designing your application pom file, add the WildFly Maven plugin `package` goal, configure it with the link:https://docs.wildfly.org/26.1/Galleon_Guide.html#wildfly-galleon-feature-packs[WildFly Galleon feature-pack]
```

Comment:
s/add the WildFly Maven plugin `package` goal/add the WildFly Maven plugin *to* the `package` goal

Reply:
I clarified that an execution of the package goal is added.